addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -402,26 +469,27 @@ const actions = {
     },
     /**
      * @param {Object} param0
-     * @param {function} param0.dispatch
-     * @param {Object} param0.getters
-     * @param {Object} param1
-     * @param {string} param1.model
-     * @param {string} param1.id
-     * @return {string}
+     * @param {Object} param0.state
+     * @param {string} chatterLocalId
      */
-    async initChatter({ dispatch, getters }, { model, id }) {
-        let threadLocalId;
-        const thread = getters.thread({ _model: model, id });
-        if (!thread) {
-            // TODO {xdu} maybe here add the name of the thread
-            threadLocalId = await dispatch('_createThread', { _model: model, id });
-        } else {
-            // TODO {xdu} uncomment me when name is supported
-            // dispatch('_updateThread', threadLocalId, { name: name } );
-            threadLocalId = thread.localId;
+    hideChatterAttachments({ state }, chatterLocalId) {
+        const chatter = state.chatters[chatterLocalId];
+        if (!chatter) {
+            return;
         }
-        dispatch('fetchThreadAttachments', threadLocalId);
-        return threadLocalId;
+        chatter.isAttachmentBoxVisible = false;
+    },
+    /**
+     * @param {Object} param0
+     * @param {Object} param0.state
+     * @param {string} chatterLocalId
+     */
+    hideChatterComposer({ state }, chatterLocalId) {
+        const chatter = state.chatters[chatterLocalId];
+        if (!chatter) {
+            return;
+        }
```

Comment:
again, assume `chatterLocalId` is valid, so remove this (same comment in the rest of this file)